### ISSUE TITLE
REPL panicked by pressing Tab key

### ISSUE THREAD

**Issue by insideable** [gluon-lang/gluon#408]
System info:
```
$ uname -a
Linux host 4.9.65-1-lts #1 SMP Fri Nov 24 13:02:16 CET 2017 x86_64 GNU/Linux

$ rustc -V
rustc 1.22.1 (05e2e1c41 2017-11-22)

$ ./target/debug/gluon -V
gluon 0.6.2
```

How to reproduce:
1. build & run gluon REPL
2. press Tab in prompt

```
 ./target/debug/gluon -i
gluon (:h for help, :q to quit)
> thread '<unnamed>' panicked at 'called `Option::unwrap()` on a `None` value', /checkout/src/libcore/option.rs:335:20
note: Run with `RUST_BACKTRACE=1` for a backtrace.
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Any', /checkout/src/libcore/result.rs:906:4
```

**Comment by insideable** [comment]
stack trace attached 
[gluon-panic.log](https://github.com/gluon-lang/gluon/files/1535047/gluon-panic.log)

**Comment by insideable** [comment]
stacktrace 
[gluon-panic.log.txt](https://github.com/gluon-lang/gluon/files/1535054/gluon-panic.log.txt)

**Comment by Marwes** [comment]
Thanks for reporting! Fixed with #409.
